### PR TITLE
Added my implementation of CNN

### PR DESCRIPTION
**Pull request by therealyubraj**
Convolutional Neural Network from scratch in JS. Example is demonstrated on MNIST dataset. 
The uploaded brain.json contains brain with 80% accuracy on test dataset.
The example can be run on browser on :
https://therealyubraj.github.io/CNN_JS/
Click on the load button to load pre-trained brain.

### CODE REVIEW

**Comment by therealyubraj** [comment]
"ci/circleci Expected — Waiting for status to be reported"
Why is this showing here????

**Comment by shiffman** [comment]
Thanks for sharing! This repo hasn't been maintained in a while so I think some of the tests / actions aren't working properly!

**Comment by therealyubraj** [comment]
Thanks for merging. :)
